hasura/migrations/default/1720894281631_alter_table_public_app_metadata_alter_column_supported_languages/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "public"."app_metadata" ALTER COLUMN "supported_languages" TYPE text[];
+alter table "public"."app_metadata" alter column "supported_languages" set default '{"en"}';
```

Comment:
```suggestion
alter table "public"."app_metadata" alter column "supported_languages" set default '["en"]';
```

Reply:
Should the default value be like this if it's an array?

Reply:
Yes that's the format for arrays